templates/desktop/index.html

```diff
@@ -1,98 +1,94 @@
-{% if hero %}
-<div id="ubuntu-kyin" class="row row-hero">
-    <div class="six-col">
-        {% include "templates/_cms-block.html" with block_content=hero %}
-        {% include "templates/_cms-block.html" with block_content=hero_cta extra_classes="cta-links" %}
-        {% include "templates/_cms-block.html" with block_content=hero_links %}
+
+<section class="p-strip--image is-deep is-bordered" style="background-image:url('/static/img/desktop/desktop-hero.jpg'); background-position:center;">
+  <div class="row">
+    <div class="col-12 suffix-6">
+      <div class="p-card--overlay">
+        <h1>Ubuntu桌面操作系统</h1>
+        <p>Ubuntu是世界最受欢迎操作系统之一，是一款完全开源，由国际软件商家Canonical作为坚实后盾的操作系统。因为它对不同硬件的支持和易用，以及它至核心的安全机制，目前在全世界有众多商业，政府及教育机构已经采用Ubuntu操作系统。</p>
+        <a href="http://cn.ubuntu.com/download">下载Ubuntu</a>
+      </div>
     </div>
-</div>
-{% endif %}
+  </div>
+</section>
 
-{# ================ Who is Ubuntu made for? ================ #}
-{% if for_you and for_business and for_developers %}
-<div id="ubuntu-kylin-user" class="row row-grey">
-    <div class="eight-col">
-        {% include "templates/_cms-block.html" with block_content=who_is_kylin_for %}
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-12">
+      <h3>适合任何人的操作系统</h3>
     </div>
-    <div class="twelve-col equal-height vertical-divider">
-        <div class="four-col for-you">
-            {% include "templates/_cms-block.html" with block_content=for_you %}
-        </div>
-        <div class="four-col for-business">
-            {% include "templates/_cms-block.html" with block_content=for_business %}
-        </div>
-        <div class="four-col last-col for-developers">
-            <div class="twelve-col">
-                {% include "templates/_cms-block.html" with block_content=for_developers extra_classes="cta-links" %}
-            </div>
-            {% include "templates/_cms-block.html" with block_content=for_developers_additional %}
-        </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h4>作为个人电脑</h4>
+      <p>既快速又安全、开源、免费，有数以千计的应用程序可供选择。从工作学习、到娱乐游戏，无所不能。</p>
+      <a class="p-link--external" href="http://tour.ubuntu.com/zh-CN/">试试在线模拟</a>
     </div>
-</div>
-{% endif %}
-
-{# ================ ubuntu kylin ============== #}
-{% if ubuntu_kylin %}
-<div class="row">
-    <div class="six-col">
-        <div class="twelve-col">
-            {% include "templates/_cms-block.html" with block_content=ubuntu_kylin extra_classes="ubuntu-kylin cta-links" %}
-        </div>
-        {% include "templates/_cms-block.html" with block_content=ubuntu_kylin_additional extra_classes="ubuntu-kylin-additional" %}
+    <div class="col-4 p-divider__block">
+      <h4>对于企业用户</h4>
+      <p>安全、直观且易于管理，可节约企业成本费用，但丝毫不影响其性能；可以大大降低企业IT硬件和运营维护的总成本。</p>
+      <a class="p-link--external" href="maito:chinamarketing@canonical.com">联络我们获得更多信息</a>
```

Comment:
We don't put external links on mailto links. Also this link should be in a `p` tag.

Reply:
This has still not been actioned.